releasenotes/releasenotes_V0.8.4.md

```diff
@@ -0,0 +1,12 @@
+# Release Notes 0.8.4
+
+## New Features
+
+- Extended common utils package(s) #281
+- Go Version 1.16 support #288
+- Added `triggeredAfter` property to `Task` structs [#3681](https://github.com/keptn/keptn/issues/3681)
+- Added structs for supporting the closed loop remediation use case introduced in Keptn v0.8.3 #287
+- Removed obsolete remediation use case structs #293
+- Added structs to support user-managed deployment URIs #289
```

Comment:
Please add links to `https://github.com/keptn/keptn/issues/` for all those issues